.circleci/config.yml

```diff
@@ -241,6 +242,16 @@ commands:
               sleep 1
             done
             echo Failed waiting for MySQL && exit 1
+      - run:
+          name: "Waiting for Redis to be ready"
+          command: |
+            for i in `seq 1 10`;
+            do
+              nc -z 127.0.0.1 6379 && echo Success && exit 0
+              echo -n .
+              sleep 1
+            done
+            echo Failed waiting for MySQL && exit 1
```

Comment:
```suggestion
            echo Failed waiting for Redis && exit 1
```

Reply:
@aidan-kwon I applied, and rebase squash to make one commit :) thank you